### PR TITLE
adding r_shell and renv_dir to the call of R scripts

### PR DESCRIPTION
**Pull request by thiagogenez**
## Description

The `linuxbrew` R binary is broken due to a missing library  (below). As a result, the analysis `ComputeJaccardIndex` fails.  

```
	libicuuc.so.68 => not found
	libicui18n.so.68 => not found
```

**Related JIRA tickets:**
There is no specific ticket for this bug. It was noticed during the competition of [ENSCOMPARASW-6396](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-6396)

## Overview of changes
The shebang of `lorentz_curve_script` and `jaccard_index_script` points to `#!/usr/bin/env Rscript` which points to the broken Linuxbrew's R.

#### Change 1
Adding the variable `r_shell` pointing to the System's Spack software management at the call.

#### Change 2
Setup the variable `renv_lib` pointing to Compara's shared space where the R modules were installed before the script Call. Otherwise, the following error will be raised

```
STDERR Error in library(ggplot2) : there is no package called âggplot2â
```

## Testing
Yes, during the competition of [ENSCOMPARASW-6396](https://www.ebi.ac.uk/panda/jira/browse/ENSCOMPARASW-6396) 

---

For code reviewers: [code review SOP](https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Code+review+SOP)

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm [hunk 1]
```diff
@@ -66,7 +66,6 @@ sub run {
     my $self = shift @_;
 
     my $all_trees = $self->param('current_tree_adaptor')->fetch_all( -TREE_TYPE => 'tree', -MEMBER_TYPE => 'protein', -CLUSTERSET_ID => 'default' );
-
```
Comment (twalsh-ebi) [inline modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm:69]: Is it necessary to delete this blank line? Making a whitespace change like this makes it more difficult to focus on the relevant changes.

**Review by twalsh-ebi — COMMENTED** [review]
Thanks @thiagogenez for this fix. Aside from my comment about extraneous whitespace changes, I just have one comment about the code changes themselves. I don't generally like to split hairs about variable names, but I think using `r_shell` for `Rscript` might cause confusion with the `R` executable, and something like `rscript_exe` might be more unambiguous.

File: modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm (hunk 1 above)
Comment (thiagogenez) [reply modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm:69]: Sorry, it is a residual change due to the tests I've performed on the code. However, the extra blank line makes the code less condensed to read.

Comment (twalsh-ebi) [reply modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm:69]: @thiagogenez, can you please revert this to how it is on the `release/111` branch ([here](https://github.com/Ensembl/ensembl-compara/blob/0858b045bccd870f8b44694df14ec93ef84f8cb6/modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/ComputeJaccardIndex.pm#L69))? You have restored the blank line, but now it contains whitespace that wasn't present before, so this is still an extraneous whitespace change.

**Comment by thiagogenez** [comment]
> Thanks @thiagogenez for this fix. Aside from my comment about extraneous whitespace changes, I just have one comment about the code changes themselves. I don't generally like to split hairs about variable names, but I think using `r_shell` for `Rscript` might cause confusion with the `R` executable, and something like `rscript_exe` might be more unambiguous.

Very good point @twalsh-ebi , thanks. I just replaced `r_shell` to `rscript_exe`.

**Review by twalsh-ebi — APPROVED** [review]
Thanks for taking my feedback on board, @thiagogenez. I'm happy to approve this PR notwithstanding the Travis failure, as it is not directly related to the changes in the PR.

**Comment by thiagogenez** [comment]
> Thanks for taking my feedback on board, @thiagogenez. I'm happy to approve this PR notwithstanding the Travis failure, as it is not directly related to the changes in the PR.

No worries, thanks a lot for the review @twalsh-ebi

**Review by Simarpreet-Kaur-Bhurji — APPROVED** [review]

**Pull request merged**